### PR TITLE
Add MultiTouchBlink examples

### LINKED ISSUE #10: Examples not enough to convey power of this library 💪
Currently, we have two great examples that helps Users to get touch points and co-ordinates from the GIGA Display Shield:
- [Touch_IRQ.ino](https://github.com/arduino-libraries/Arduino_GigaDisplayTouch/blob/4101b8a1f4a1d047228fafb36b6fd203c7481e23/examples/Touch_IRQ/Touch_IRQ.ino)
- [Touch_Polling.ino](https://github.com/arduino-libraries/Arduino_GigaDisplayTouch/blob/4101b8a1f4a1d047228fafb36b6fd203c7481e23/examples/Touch_Polling/Touch_Polling.ino)

These two examples are are a good starting point, but are limited in scope. Specifically we need examples that:
- Show how to detach and re-attach interrupts. See https://github.com/arduino-libraries/Arduino_GigaDisplayTouch/issues/8
- Can be run independently of the Serial Monitor.
- Functions without depending on the [arduino-libraries/Arduino_GigaDisplay](https://github.com/arduino-libraries/Arduino_GigaDisplay) library.

### PR DESCRIPTION
**Pull request by aliphys**
This PR adds two examples, that blinks the LED on the GIGA R1 WiFi based on the number of touches registered on the screen. 
- ⁉️ MultiTouchBlink_IRQ.ino shows how to attach and detach an interrupt (be calling `touchDetector.begin();`
- 🚥 The LED onboard the GIGA R1 WiFi is used to display the feedback from the touch events. Using the RGB LED on the GIGA Display Shield itself would create a cyclical dependency since the [GIGADisplayRGB](https://github.com/arduino-libraries/Arduino_GigaDisplay/blob/6536ab589056b343705c030c2ea1e21a986e1a36/src/GigaDisplayRGB.h) class is part of [arduino-libraries/Arduino_GigaDisplay](https://github.com/arduino-libraries/Arduino_GigaDisplay)
- 🌐 Both interrupt `MultiTouchBlink_IRQ.ino` and polling approaches `MultiTouchBlink_Polling.ino` are covered. 

✨ Thanks for initial testing and feedback by @Hannes7eicher and @pedromsousalima
Closes https://github.com/arduino-libraries/Arduino_GigaDisplayTouch/issues/10

### CODE REVIEW

File: examples/MultiTouchBlink_IRQ/MultiTouchBlink_IRQ.ino [hunk 1]
```diff
@@ -0,0 +1,61 @@
...
+
+#include "Arduino_GigaDisplayTouch.h"
+
+Arduino_GigaDisplayTouch touchDetector;
+
+int contactPoints = 0;
+
+void gigaTouchHandler(uint8_t contacts, GDTpoint_t* points) {
+  contactPoints = contacts;
+  touchDetector.end();  // close the touch controller
```
Comment (leonardocavagnis) [inline examples/MultiTouchBlink_IRQ/MultiTouchBlink_IRQ.ino:30]:
```suggestion
  touchDetector.detach(); 
```

File: examples/MultiTouchBlink_IRQ/MultiTouchBlink_IRQ.ino [hunk 2]
```diff
@@ -0,0 +1,61 @@
...
+  if (contactPoints > 0) {
+    Serial.println("Contact Detected! Will Blink " + String(contactPoints) + " times!");
+    for (int i = 0; i < contactPoints; i++) {
+      digitalWrite(LED_BUILTIN, LOW);
+      delay(250);
+      digitalWrite(LED_BUILTIN, HIGH);
+      delay(250);
+    }
+    contactPoints = 0;
+    touchDetector.begin();
```
Comment (leonardocavagnis) [inline examples/MultiTouchBlink_IRQ/MultiTouchBlink_IRQ.ino:57]:
```suggestion
```

File: examples/MultiTouchBlink_Polling/MultiTouchBlink_Polling.ino [hunk 3]
```diff
@@ -0,0 +1,62 @@
...
+void setup() {
+  Serial.begin(115200);
+  while (!Serial) {}
+
+  if (touchDetector.begin()) {
+    Serial.println("Touch controller init - OK");
+  } else {
+    Serial.println("Touch controller init - FAILED");
+    while (1)
+      ;
```
Comment (leonardocavagnis) [inline examples/MultiTouchBlink_Polling/MultiTouchBlink_Polling.ino:38]:
```suggestion
```

File: examples/MultiTouchBlink_Polling/MultiTouchBlink_Polling.ino [hunk 4]
```diff
@@ -0,0 +1,62 @@
...
+
+void setup() {
+  Serial.begin(115200);
+  while (!Serial) {}
+
+  if (touchDetector.begin()) {
+    Serial.println("Touch controller init - OK");
+  } else {
+    Serial.println("Touch controller init - FAILED");
+    while (1)
```
Comment (leonardocavagnis) [inline examples/MultiTouchBlink_Polling/MultiTouchBlink_Polling.ino:37]:
```suggestion
    while (1) ;
```

**Review by leonardocavagnis — COMMENTED** [review]
Instead of calling `begin`/`end`, it might make more sense to use the `detach` function.

**Comment by aliphys** [comment]
The `MultiTouchBlink_Polling.ino` example has an issue where sends the same message to the Serial twice, for a single touch event. It seems as if the values provided by `touchDetector.getTouchPoints(points);` are not flushed even with a one second delay.
https://github.com/arduino-libraries/Arduino_GigaDisplayTouch/blob/e1a53e2e68755a239559b27cf8eb17ad06acb648/examples/MultiTouchBlink_Polling/MultiTouchBlink_Polling.ino#L47

##  `MultiTouchBlink_IRQ.ino`
✅ Each touch event, is represented by a single message in the Serial Monitor (as well as a single Blink of the GIGA R1 WiFi LED)

https://github.com/arduino-libraries/Arduino_GigaDisplayTouch/assets/75624145/1d51967a-2d47-492e-90ec-9e8f2ae5651a

##  `MultiTouchBlink_Polling.ino`
❌ Each touch event, is represented by a two messages in the Serial Monitor, separated by the delay value (as well as a single Blink of the GIGA R1 WiFi LED)



https://github.com/arduino-libraries/Arduino_GigaDisplayTouch/assets/75624145/b0a4ac30-9fcb-4bf3-a347-db785be1666d

File: examples/MultiTouchBlink_IRQ/MultiTouchBlink_IRQ.ino (hunk 1 above)
Comment (aliphys) [reply examples/MultiTouchBlink_IRQ/MultiTouchBlink_IRQ.ino:30]: I did not see a member function for `.detach()` in the [Arduino_GigaDisplayTouch.h](https://github.com/arduino-libraries/Arduino_GigaDisplayTouch/blob/main/src/Arduino_GigaDisplayTouch.h) header. When making the change recommended the following error is given.

```
C:\GitHub\Arduino\Arduino_GigaDisplayTouch\examples\MultiTouchBlink_IRQ\MultiTouchBlink_IRQ.ino:30:17: error: 'class Arduino_GigaDisplayTouch' has no member named 'detach'
   touchDetector.detach();  // close the touch controller
                 ^~~~~~

Using library Arduino_GigaDisplayTouch at version 1.0.1 in folder: C:\Users\Ali Jahangiri\Documents\Arduino\libraries\Arduino_GigaDisplayTouch 
Using library Wire in folder: C:\Users\Ali Jahangiri\AppData\Local\Arduino15\packages\arduino\hardware\mbed_giga\4.1.3\libraries\Wire (legacy)
exit status 1

Compilation error: 'class Arduino_GigaDisplayTouch' has no member named 'detach'
```

I agree, having a `detach()` method would be greatly appreciated 😄 ! See https://github.com/arduino-libraries/Arduino_GigaDisplayTouch/issues/8

File: examples/MultiTouchBlink_IRQ/MultiTouchBlink_IRQ.ino (hunk 2 above)
Comment (aliphys) [reply examples/MultiTouchBlink_IRQ/MultiTouchBlink_IRQ.ino:57]: I agree it is not a `pretty` solution, but it works. One should not use `begin()` in the middle of a sketch to restart the GT911 module. See: https://github.com/arduino-libraries/Arduino_GigaDisplayTouch/pull/11#discussion_r1637927127
